app/theme/client/imports/components/sidebar/sidebar.css

```diff
@@ -64,7 +64,9 @@
 	}
 
 	& .unread-rooms {
-		display: none;
+		padding: calc(var(--sidebar-small-default-padding) - 8px);
```

Comment:
What are those 8 pixels? scrollbar?

Reply:
No, the `.sidebar-item` has 3px padding top and the total height is 34px (base on `var(--sidebar-item-height-medium)`). 
So I put the default padding there plus 8px to equal the `sidebar-item`